dl/src/main/scala/com/intel/analytics/bigdl/dataset/Types.scala

```diff
@@ -102,6 +104,7 @@ class Sample[T: ClassTag] (
 
   def copyToLabel(storage: Array[T], offset: Int, length: Int): Unit = {
```

Comment:
copyToLabel  and copyToFeature will not change the featureTensor and labelTensor. They just copy the featureTensor or labelTensor out.